app/services/support_interface/user_permissions_export.rb

```diff
@@ -0,0 +1,61 @@
+module SupportInterface
+  class UserPermissionsExport
```

Comment:
As a side note as it's out of scope of this PR and I know it originated from the info on the Trello cards,
but I think a “nice to have” would be to unify the names of those provider permissions exports to make them a bit more descriptive. We now have:
- Active provider users
- Active provider user permissions
- Provider access controls
- User permissions
- Organisation permissions

I will add a card to the backlog.